### PR TITLE
[15.0][mig] event quick registration

### PR DESCRIPTION
**Pull request by damdam-s**
add event_quick_registration

### CODE REVIEW

File: event_quick_registration/wizard/event_quick_registration.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    event_ticket_id = fields.Many2one("event.event.ticket", required=True)
+    qty = fields.Integer(required=True)
+    confirm_registration = fields.Boolean()
+    email = fields.Char()
+    phone = fields.Char()
+    name = fields.Char()
+
+    @api.model
+    def default_get(self, fields):
+        res = super(EventQuickRegistration, self).default_get(fields)
```
Comment (jcoux) [inline event_quick_registration/wizard/event_quick_registration.py:21]:
```suggestion
        res = super().default_get(fields)
```

File: event_quick_registration/wizard/event_quick_registration.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            "email": self.email,
+            "origin": _("Quick creation"),
+        }
+
+    def create_attendees(self):
+        self.ensure_one()
+        Registration = self.env["event.registration"]
+        attendees = Registration.browse()
+        for _i in range(self.qty):
+            attendees |= Registration.create(self._get_registration_data())
```
Comment (jcoux) [inline event_quick_registration/wizard/event_quick_registration.py]: Seems `+=` is more quickly! (Thanks @ivantodorovich )
```suggestion
            attendees += Registration.create(self._get_registration_data())
```

**Review by jcoux — APPROVED** [review]
2 little suggestions: pre-approved

File: event_quick_registration/wizard/event_quick_registration.py (hunk 2 above)
Comment (ivantodorovich) [inline event_quick_registration/wizard/event_quick_registration.py]:
```suggestion
        attendees = self.env["event.registration"].create(
            [self._get_registration_data() for i in range(self.qty)]
        )
```
something like tihs is even quicker :)
leverage model_create_multi

**Review by ivantodorovich — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**